developer-handbook/github-process/pull-request-process.md

```diff
@@ -0,0 +1,36 @@
+# Pull Request creation
+
+Here are recommendations to consider from the moment you create a new branch to work on.
+
+## Pre-creation considerations
+
+**Consider creating your Pull Request when you're 30-50% through**
+_If it's a big piece_, consider creating your PR when you're 30-50% through. Create it as a draft, so no one gets notified of every commit. 
```

Comment:
What about suggesting creation in draft "as soon as possible", with a focus on filling out the PR description up front? 

PRs flag that work is being done and provide context for other developers. Short, contextless descriptions created after most of the work are done can feel like they make things quicker, but potentially disadvantage anyone trying to review, or understand the work later.

Reply:
How about "Consider creating your Pull Request as soon as possible. Think of PRs less as validation of work done and more as visibility of work in progress. The description ideally offers context as to why the work is being done as well as explaining how the solution was decided on, which can be very helpful to reviewers. Create it as a draft, so no one gets notified of every commit."

Reply:
Love that, thank you!

Reply:
I've made changes to the whole section, [see here](https://github.com/wellcomecollection/developer-handbook/pull/1/files#diff-98802f63a1e1ae0bb73422d2c806ebd85275584be6ecf804d2b03e002481b74dR7-R12)